src/api/db/migrate/20211110142710_create_workflow_runs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkflowRuns < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflow_runs, id: :integer do |t|
+      t.text :request_headers
+      t.text :request_payload
+      t.string :status
```

Comment:
Having an [enum](https://api.rubyonrails.org/classes/ActiveRecord/Enum.html) for `status` would be better to limit this to only a set of values. The column would then be a tinyint (so integer with `limit: 1` in the migration) instead of a string.

Reply:
Sure. Changed.